src/service/api/article.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+const {Router} = require(`express`);
+const {HttpCode, NOT_FOUND_TEXT} = require(`../../const`);
+const articleValidator = require(`../middlewares/article-validator`);
+const articleExist = require(`../middlewares/article-exist`);
+const commentValidator = require(`../middlewares/comment-validator`);
+
+const route = new Router();
+
+module.exports = (app, articleService, commentService) => {
+  app.use(`/articles`, route);
```

Comment:
```suggestion

module.exports = (app, articleService, commentService) => {
const route = new Router();
  app.use(`/articles`, route);
```
так нужно на всех api сделать, тк дальше могут быть странные сайд эффеккты